src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+public class Main {
+    public static void main(String[] args) {
+        Figure[] figures = new Figure[4];
+        for (int a = 0; a < figures.length; a++) {
+            if (a <= figures.length / 2) {
+                figures[a] = FigureSupplier.getRandomFigure();
+                figures[a].color = ColorSupplier.getRandomColor();
+                figures[a].area = figures[a].getArea();
```

Comment:
You should set `color` inside getRandomFigure method, when you create figure

Reply:
`area` should be calculated. It's not necessary to have field `area` in `Figure` class